qf/submission.go

```diff
@@ -42,6 +42,17 @@ func (s *Submission) GetStatusByUser(userID uint64) Submission_Status {
 	return Submission_NONE
 }
 
+// SetReleasedAndMarks sets the submission's released status, grades and score
+func (s *Submission) SetReleasedAndMarks(request *UpdateSubmissionRequest) {
```

Comment:
We don't have a concept `Marks` so this is confusing.
```go
// SetGradesAndRelease sets the submission's grade, score and released status.
```

Also, we need a test for this.